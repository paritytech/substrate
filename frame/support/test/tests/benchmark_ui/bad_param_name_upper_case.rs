use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;

#[benchmarks]
mod benches {
	#[benchmark]
	fn bench(D: Linear<1, 2>) {
		#[block]
		{}
	}
}

fn main() {}
