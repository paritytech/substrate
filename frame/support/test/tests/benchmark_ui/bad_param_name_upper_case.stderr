error: Benchmark parameter names must consist of a single lowercase letter (a-z) and no other characters.
 --> tests/benchmark_ui/bad_param_name_upper_case.rs:8:11
  |
8 |     fn bench(D: Linear<1, 2>) {
  |              ^
